content/en/docs/Concepts/crds/installplan.md

```diff
@@ -3,6 +3,63 @@ title: "InstallPlan"
 weight: 5
 ---
 
-{{% alert title="Warning" color="warning" %}} This page is under construction{{% /alert %}}
-
 An InstallPlan defines a set of resources to be created in order to install or upgrade to a specific version of a ClusterService defined by a CSV.
+
+Operator authors are responsible for defining which `InstallModes` their operator supports within its ClusterServiceVersion (CSV). There are four `InstallModes` that an operator can support:
+* `OwnNamespace`: If supported, the operator can be configured to watch for events in the namespace it is deployed in.
+* `SingleNamespace`: If supported, the operator can be configured to watch for events in a single namespace that the operator is not deployed in.
+* `MultiNamespace`: If supported, the operator can be configured to watch for events in more than one namespace.
+* `AllNamespaces`: If supported, the operator can be configured to watch for events in all namespaces.
+
+>Note: If a CSV's spec omits an entry of InstallModeType, that type is considered unsupported unless support can be inferred by an existing entry that implicitly supports it.
+
+Cluster admins cannot override which `InstallMode`s an operator supports, and so should understand how to create an `OperatorGroup` that supports each `InstallMode`. Let's look at an example of an `OperatorGroup` implementing each type of `InstallMode`:
+
+
+# OwnNamespace
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: own-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - own-namespace
```

Comment:
I think we should not use `own-namespace`, it looks like it's a special namespace since it matches `OwnNamespace`. Not knowing anything I would think that it has to be called `own-namespace`.  Maybe using something like `the-operators-namespace` makes it more generic.